testing/mm/mm_main.c

```diff
@@ -119,11 +119,23 @@ static const int g_alignment2[NTEST_ALLOCS / 2] =
 
 static FAR void       *g_allocs[NTEST_ALLOCS];
 static struct mallinfo g_alloc_info;
+int allow_malloc_faill = 0;
```

Comment:
why need this flag, @chenzihan0416 ?

Reply:
This flag has been deleted and the first submission requires the maintainer's help to trigger CI, thank you! @xiaoxiang781216